pkg/kubernetes/deploy.js

```diff
@@ -0,0 +1,454 @@
+    var SERVICE = "Service";
+    var RC = "ReplicationController";
+    var NS = "Namespace";
+    var valid_manifest = false;
+    var valid_ns = false;
+    var namespacetxt = "";
+
+
+    function failure(ex) {
+        console.warn(ex);
+    }
+
+    function isJsonString(str) {
+        try {
+            JSON.parse(str);
+        } catch (e) {
+            return false;
+        }
+        return true;
+    }
+
+    function deploy_manager() {
+
+        var is_deploying = $('#deploy-app-deploying');
+
+        function deploy(namespace, jsonData) {
+
+            remove_notifications();
+
+            var services = [];
+            var rcs = [];
+            var pods = [];
+            var namespaces = [];
+            var has_errors = false;
+            var text = "";
+            var file_note = $('#deploy-app-manifest-file-note');
+            var file_note_details = $('#deploy-app-manifest-file-note-details');
+            var deploying_app_details = $('#deploy-app-deploying-details');
+            show_progress_message();
+
+            var deploy_btn = $('#deploy-app-start');
+            deploy_btn.prop("disabled", true);
+
+            if (isJsonString(jsonData)) {
+                var jdata = JSON.parse(jsonData);
+                if(jdata.kind === "List") {
+                    if (jdata.items) {
+                        for (var i = 0; i < jdata.items.length; i++) {
+                            var ent_json = jdata.items[i];
+                            if (ent_json.kind === SERVICE) {
+                                services.push(ent_json);
+                            } else if (ent_json.kind === POD) {
+                                pods.push(ent_json);
+                            } else if (ent_json.kind === RC) {
+                                rcs.push(ent_json);
+                            } else if (ent_json.kind === NS) {
+                                namespaces.push(ent_json);
+                            }
+                        }
+                    } else {
+                        text = _("Unable to Read the file.Please check the json file. ");
+                        file_note.show().text(text);
+                        return;
+                    }
+                } else {
+                    if (jdata.kind === SERVICE) {
+                        services.push(jdata);
+                    } else if (jdata.kind === POD) {
+                        pods.push(jdata);
+                    } else if (jdata.kind === RC) {
+                        rcs.push(jdata);
+                    } else if (jdata.kind === NS) {
+                        namespaces.push(jdata);
+                    } else {
+                        text = _("Unsupported Entity. ");
+                        file_note.show().text(text);
+                        return;
+                    }
+                }
+            } else {
+                text = _("Unable to Read the file.Please check the json file. ");
```

Comment:
Please change this text: "Unable to read the Kubernetes application manifest file."